services/wallet/api.go

```diff
@@ -137,6 +137,10 @@ func (api *API) GetWalletToken(ctx context.Context, addresses []common.Address)
 	return api.reader.GetWalletToken(ctx, clients, addresses, currency)
 }
 
+func (api *API) GetLastWalletTokenUpdate() map[common.Address]time.Time {
```

Comment:
we tend to use unix timestamps everywhere. What does a `time.Time` look like in JSON?

Reply:
fixed